src/Helpers/TagsHelper.cs

```diff
@@ -39,6 +41,23 @@ public static ContentReference GetContentReference(Guid contentGuid)
             return PermanentLinkUtility.FindContentReference(contentGuid) ?? ContentReference.EmptyReference;
         }
 
+        public static IEnumerable<ContentReference> GetContentReferences(List<Guid> contentLinks)
```

Comment:
Can this be replaced by ContentLoader.Items?

Reply:
You mean ContentLoader.GetItems? Can't seem to find any method which accepts an collection of Guids

Reply:
A short way for the same logic, what do you think?

```suggestion
        public static IEnumerable<ContentReference> GetContentReferences(List<Guid> contentLinks)
        {
            foreach (var contentLink in contentLinks)
            {
                var reference = GetContentReference(contentLink);

                if (!ContentReference.IsNullOrEmpty(reference))
                {
                    yield return reference;
                }
            }
        }
```